src/components/Tooltip/Tooltip.js

```diff
@@ -56,13 +59,27 @@ class Tooltip extends PureComponent {
     window.removeEventListener('scroll', this.hideTooltip, false);
   }
 
+  toggleTooltip() {
+    if (this.state.visible) {
+      this.hideTooltip();
+    } else {
+      this.showTooltip();
+    }
+  }
+
   render() {
     const { props, state } = this;
 
     return (
-      <span // a span so it's valid HTML no matter where it's used
+      // a span so it's valid HTML no matter where it's used
+      // Note:
+      //   the onClick there for touch devices
+      //   but it doesn't seem right to apply role=button
+      //   hence the eslint disabling
+      <span // eslint-disable-line
         onMouseOver={this.showTooltip}
         onMouseOut={this.hideTooltip}
+        onClick={this.toggleTooltip}
```

Comment:
Minor: this has the slightly strange behaviour that if you click on desktop, the tooltip will show as soon as you hover, then hide when you click. Not a big deal, but you could instead do:

```js
onTouchStart={this.toggleTooltip}
```

Reply:
Yeah I noted that but didn't really think it was a negative thing. Happy to change to that event though.